hestia-sat/src/payload.rs

```diff
@@ -43,13 +45,24 @@ pub struct Config {
     
     /// Installation directory, used for uts-update
     pub install_path: Option<String>,
+
+    /// Send error logging to syslog instead of console
+    #[serde(default)]
+    pub syslog: bool,
 }
 
 impl Config {
     pub fn read() -> Config {
         dotenv().ok();
-        let _ = env_logger::try_init(); // don't fail if called multiple times
-        envy::prefixed("UTS_").from_env().unwrap()
+        let config: Config = envy::prefixed("UTS_").from_env().unwrap();
+        if config.syslog {
+            syslog::init(Facility::LOG_USER, LevelFilter::Info, None)
```

Comment:
Does `syslog::init()` fail if called multiple times? I forget where `Config::read()` can be called multiple times, but I guess it must happen sometimes because of the comment in the `else` block below.

Reply:
It returns an error result, but it’s otherwise harmless: the first init sticks. I’ll probably add an explicit discard of the result to make this more intentional.